README.md

```diff
@@ -62,7 +36,7 @@ Follow the [simple-server instructions](https://github.com/simpledotorg/simple-s
 
 #### Java 8's date and time
 
-Clinic uses [lazythreetenbp](https://github.com/gabrielittner/lazythreetenbp) for working with date and time. Due to some limitations, the IDE does not know how to download its sources. As a work around, the sources can be downloaded from [the maven repository](http://search.maven.org/#search%7Cga%7C1%7Cthreetenbp) and manually attached to Android Studio.
+Simple uses [lazythreetenbp](https://github.com/gabrielittner/lazythreetenbp) for working with date and time. Due to some limitations, the IDE does not understand how to download its sources. As a work around, the sources can be downloaded from [the maven repository](http://search.maven.org/#search%7Cga%7C1%7Cthreetenbp) and manually attached to Android Studio.
```

Comment:
Let's not mix styles -- either use `we` everywhere or `Simple` everywhere.

Reply:
Actually, on second thought, we should remove the entire Miscellaneous section from the README, and move them into the code as comments.
- lazythreetenbp info can be mentioned in the code as a comment 
- workmanager debugging info can be mentioned as a comment 
- androidTest failure workaround can also be mentioned somewhere as a comment. Maybe in the base-class of the android tests?

Reply:
Comments in the code would have been nice, but dates are used everywhere in the app. There is no starting point. Same for `androidTest`. 

We can move these to separate wiki pages.

Reply:
How about the place where lazythreetenbp is being initialised?

Reply:
Hrm, if you're a new developer and you encounter one of the date classes, will you be able to figure out that you should open the place where `lazythreetenbp` is being initialized?

Reply:
No, but if you're a new developer you'd be too inexperienced to read any documentation or follow anything. An experienced developer will just grep their way through things!

Reply:
Sorry I meant "new to this project".

Even as an experienced dev, it took me a while to understand why am I unable to add sources. Had to go through some pages on Github and some other issue tracking website. I don't think it has to do anything with your experience.